stylesheets/SimpliGov-basic.css

```diff
@@ -133,7 +133,7 @@ div#workflow-background-wrapper {
 }
 
 /* Hide field value */
-#element108, #element123, #element138 {
+#element108, #element123, #element138, #element180, #element183, #element184 {

```

Comment:
Up to you whether this sounds like a good idea, but I could imagine it might be useful down the road to add a comment for each of these describing in human terms what they apply to. Something along the lines of:

    /* Birthday field */
    #element1,
    /* Zipcode field */
    #element2 {

Reply:
I think this is a really good idea. I'll add a ticket to the backlog to make this change in either next sprint or the following one

Reply:
Update: Asana ticket added to backlog here: https://app.asana.com/0/1170867711449497/1200845867325835/f